frontend/src/components/input/editor/mentionSuggestion.ts

```diff
@@ -0,0 +1,190 @@
+import {VueRenderer} from '@tiptap/vue-3'
+import {computePosition, flip, shift, offset, autoUpdate} from '@floating-ui/dom'
+import type {Editor} from '@tiptap/core'
+
+import MentionList from './MentionList.vue'
+import ProjectUserService from '@/services/projectUsers'
+import {fetchAvatarBlobUrl, getDisplayName} from '@/models/user'
+import type {IUser} from '@/modelTypes/IUser'
+
+interface MentionItem {
+	id: number
+	label: string
```

Comment:
Since this will always be the user's display name, we might as well call it like that.